t/t2205-job-info-security.t

```diff
@@ -69,6 +78,28 @@ test_expect_success 'flux job eventlog fails (wrong user)' '
         unset_userid
 '
 
+test_expect_success 'flux job eventlog works (owner, inactive)' '
+        jobid=$(submit_job) &&
+        move_inactive $jobid &&
```

Comment:
Same comment here - need to wait for job manager to finish appending to eventlog?